app/containers/HomeContainer/index.js

```diff
@@ -37,89 +46,112 @@ const Container = styled.div`
     padding: ${props => props.padding}px;
   }
 `;
-const RightContent = styled.div`
-  display: flex;
-  align-self: flex-end;
-`;
+
 export function HomeContainer({
-  dispatchGithubRepos,
-  dispatchClearGithubRepos,
+  dispatchSongs,
+  dispatchClearSongs,
   intl,
-  reposData = {},
-  reposError = null,
-  repoName,
+  itunesData,
+  itunesError,
+  artistName,
   maxwidth,
   padding
 }) {
   useInjectSaga({ key: 'homeContainer', saga });
   const [loading, setLoading] = useState(false);
+  const [isPlaying, setIsPlaying] = useState(false);
+  const [tuneId, setTuneId] = useState(null);
+  const [audio, setAudio] = useState(null);
+  const [curTime,setCurTime] = useState(0)
+  const [trackUrl,setTrackUrl]=useState(null)
 
   useEffect(() => {
-    const loaded = get(reposData, 'items', null) || reposError;
+    const loaded = get(itunesData, 'results', null) || itunesError;
     if (loading && loaded) {
       setLoading(false);
     }
-  }, [reposData]);
+  }, [itunesData]);
 
   useEffect(() => {
-    if (repoName && !reposData?.items?.length) {
-      dispatchGithubRepos(repoName);
+    if (artistName && !itunesData) {
+      dispatchSongs(artistName);
+      console.log(dispatchSongs('abcd'));
       setLoading(true);
     }
   }, []);
-
   const history = useHistory();
 
   const handleOnChange = rName => {
     if (!isEmpty(rName)) {
-      dispatchGithubRepos(rName);
+      dispatchSongs(rName);
       setLoading(true);
     } else {
-      dispatchClearGithubRepos();
+      dispatchClearSongs();
     }
   };
   const debouncedHandleOnChange = debounce(handleOnChange, 200);
 
-  const renderRepoList = () => {
-    const items = get(reposData, 'items', []);
-    const totalCount = get(reposData, 'totalCount', 0);
+  const playAudio = (url, trackId) => {
+    
+    setIsPlaying(true);
+    setTuneId(trackId);
+    audio?.pause();
+    let curAudio = new Audio(url);
+    setAudio(curAudio);
+   if(curTime>0 && url===trackUrl){
```

Comment:
instead of `curTime > 0 && url === trackUrl`, you can use `curTime && url === trackUrl` since values above 0 will evaluate to truthy